types/default-theme.d.ts

```diff
@@ -148,7 +148,25 @@ export namespace DefaultTheme {
 
   export type SidebarItem =
     | { text: string; link: string }
-    | { text: string; link?: string; items: SidebarItem[] }
+    | {
+        text: string
+        link?: string
+        items: SidebarItem[]
+
+        /**
+         * If `true`, toggle button is shown.
+         *
+         * @default false
+         */
+        collapsible?: boolean
+
+        /**
+         * If `true`, collapsible group is collapsed by default.
+         *
+         * @default false
+         */
+        collapsed?: boolean
```

Comment:
can't we only have this property? and if collapsed != undefined we assume the toggle should show?, this way we don't need the collapsible property. 
I know we already have collapsible but collapsed? should cover all options with that logic. 
Does that makes sense @kiaking?

Reply:
What if the user would want to accomplish collapsible and expanded initially?

Reply:
@LittleSound 

```
collapsed == null -> not collapsible
collapsed === true -> collapsible but collapsed
collapsed === false -> collapsible but expanded
```

Reply:
I see. However, I ended up here like this by following the configuration design pattern that already existed. Modifying the configuration property is not the purpose of this PR, maybe it should be made as a separate issue?

Reply:
> Modifying the configuration property is not the purpose of this PR

Ah don't worry, it wasn't specific to your PR. It needs to be done at top-level.